### ISSUE TITLE
Tests fail on big-endian

### ISSUE THREAD

**Issue by opoplawski** [python-xlib/python-xlib#76]
On ppc64 I get lots of errors (after applying PR #75) like this:
```
ERROR: testUnpackReply0 (test.test_requests_be.TestAllocColor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/fedora/rpmbuild/BUILD/python-xlib-0.19/test/test_requests_be.py", line 3829, in testUnpackReply0
    args, remain = request.AllocColor._reply.parse_binary(self.reply_bin_0, dummy_display, 1)
  File "/home/fedora/rpmbuild/BUILD/python-xlib-0.19/Xlib/protocol/rq.py", line 1146, in parse_binary
    val = struct.unpack(self.static_codes, data[:self.static_size])
TypeError: a bytes-like object is required, not 'str'

======================================================================
FAIL: testPackReply0 (test.test_requests_be.TestAllocColor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/fedora/rpmbuild/BUILD/python-xlib-0.19/test/test_requests_be.py", line 3824, in testP
ackReply0
    assert bin == self.reply_bin_0
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fedora/rpmbuild/BUILD/python-xlib-0.19/test/test_requests_be.py", line 3826, in testP
ackReply0
    raise AssertionError(tohex(bin))
AssertionError: ["b'01002a9800000000ab080c6238bd0000'\n", "b'3fa738c8000000000000000000000000'\n"]

======================================================================
FAIL: testPackRequest0 (test.test_requests_be.TestAllocColor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/fedora/rpmbuild/BUILD/python-xlib-0.19/test/test_requests_be.py", line 3806, in testPackRequest0
    assert bin == self.req_bin_0
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fedora/rpmbuild/BUILD/python-xlib-0.19/test/test_requests_be.py", line 3808, in testPackRequest0
    raise AssertionError(tohex(bin))
AssertionError: ["b'540000041f31c7dd9b2dc2bec1ac0000'\n"]

======================================================================
FAIL: testPackReply0 (test.test_requests_be.TestAllocColorCells)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/fedora/rpmbuild/BUILD/python-xlib-0.19/test/test_requests_be.py", line 3966, in testPackReply0
    assert bin == self.reply_bin_0
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fedora/rpmbuild/BUILD/python-xlib-0.19/test/test_requests_be.py", line 3968, in testPackReply0
    raise AssertionError(tohex(bin))
AssertionError: ["b'0100d309000000140011000300000000'\n", "b'00000000000000000000000000000000'\n", "b'294f7e294d2e493558235e3f6a369bd0'\n", "b'55f601676296189d2ef21d333e931239'\n", "b'69bc1c9c64ad40d260f35ed10a97f6cb'\n", "b'2766c36a7573969e7259c77b290a45d8'\n", "b'02724b732bbed7d53cfe7f342384b117'\n"]
```
though too many to list all.

**Comment by benoit-pierre** [comment]
I believe re-generating the big-endian tests as mentioned in #75 should fix those.

**Comment by benoit-pierre** [comment]
Fixed by #77.
